### PR TITLE
add chakra ui docs extension

### PR DESCRIPTION
**Pull request by pgvr**
## Description

This is a new extension that provides short links to the Chakra UI documentation. This saves me some time instead of opening the website first and then searching.

## Type of change

- New extension

## Screencast

image

image

## Checklist

- [X] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [X] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [X] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)

### CODE REVIEW

File: extensions/chakra-ui-docs/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "chakra-ui-docs",
+  "title": "Chakra UI Docs",
```
Comment (pernielsentikaer) [inline extensions/chakra-ui-docs/package.json:4]:
```suggestion
  "title": "Chakra UI Documentation",
```

File: extensions/chakra-ui-docs/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "description": "Provide quick links to the Chakra UI component documentation.",
+  "icon": "chakra.png",
+  "author": "pgvr",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Browse Components",
```
Comment (pernielsentikaer) [inline extensions/chakra-ui-docs/package.json:12]:
```suggestion
      "title": "Browse Components",
      "subtitle": "Chakra UI",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @pgvr 👋 

Thanks for your contribution 💯 I left a few suggestions for you to address 💪 

I have some feedback:
- I think the need of UUID is overkill, you could use the title as key instead
- Remove the .svg icon, since it's not in use

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by pgvr** [comment]
Hi @pernielsentikaer, I'm sorry about the radio silence. Thank you for your feedback! Let me know whether there is anything else.

**Review by pernielsentikaer — APPROVED** [review]
Hi @pgvr 👋 

Awesome, thanks for the correction 💯 

We're ready to move to the next stage, awesome! Good job, mate 💪

**Pull request merged**